certified-connectors/DeskDirector Tech/apiProperties.json

```diff
@@ -0,0 +1,57 @@
+{
+	"properties": {
+		"connectionParameters": {
```

Comment:
Could you please update the apiProperties.json file as per the below schema? Additionally, kindly update the file with proper indentation. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/schemas/paconn-apiProperties.schema.json